cli/src/main/scala/com/advancedtelematic/tuf/cli/Cli.scala

```diff
@@ -96,6 +96,7 @@ object Cli extends App with VersionInfo {
     opt[RepoName]("repo").abbr("r").required().action { (name, c) =>
       c.copy(repoName = name)
     }
+    .text("required for all subcommands")
```

Comment:
Kinda, in theory not needed for "help", so currently just running the tool without a subcommand complains that "--repo" is missing instead of showing you the help text, if you run `garage-sign --repo repo` it shows you the help text.